changes-set.txt

```diff
@@ -99,6 +99,7 @@ make a github issue.)
 
 DONE:
 Date      Old       New         Notes
+ 6-Sep-24 difid     difidALT    swapped proofs and credits
```

Comment:
not technically needed since the statements are the same, so there are no conflicts

Reply:
Good. I'm not a fan of excessive bureaucracy, so I prefer to keep things simple when possible. I removed it.